shell/common/fixtures/shell_test.dart

```diff
@@ -270,3 +270,8 @@ void onBeginFrameWithNotifyNativeMain() {
   };
   notifyNative();
 }
+
+@pragma('vm:entry-point')
+void frameCallback(_Image, int) {
+  print('called back');
```

Comment:
Please remove the print

Reply:
(and consider adding a comment about why this method is empty and what test is using it)

Reply:
done